include/session/config/base.hpp

```diff
@@ -988,14 +1077,32 @@ class ConfigBase : public ConfigSig {
 
     /// API: base/ConfigBase::current_hashes
     ///
-    /// The current config hash(es); this can be empty if the current hash is unknown or the current
-    /// state is not clean (i.e. a push is needed or pending).
+    /// The hashes of the current config state.  This can be empty if the current hashes are not
+    /// known (i.e. a push is still in progress) or the current state is not clean (i.e. a push is
+    /// needed or pending).
+    ///
+    /// See also active_hashes(), which you often want instead of this.
+    ///
+    /// Inputs: None
+    ///
+    /// Outputs:
+    /// - `std::unordered_set<std::string>` -- Returns current config hashes
+    const std::unordered_set<std::string>& current_hashes() const;
```

Comment:
should we make this protected? I feel like we won't need this anymore for the wrappers but only the active_hashes(), and it would be good to not "use it by mistake".
Or maybe we should rename it, so we are all aware of that change when this PR gets merged

Reply:
I was hoping the type change would be enough for that, but you're right, it might not be.

I've renamed it to `curr_hashes` to induce such breakage and, hopefully, they see the comment about using `active_hashes` instead.